web/src/components/PoiFilters.tsx

```diff
@@ -60,8 +62,11 @@ const StyledButton = styled(TextButton)`
 `
 
 const StyledIcon = styled(Icon)`
-  width: 24px;
-  height: 24px;
+  min-width: 24px;
+`
+
+const StyledCheckbox = styled(Checkbox)`
+  min-width: 20px;
 `
```

Comment:
:wrench: Sadly this does not take effect yet because
a) No classname is passed in `Checkbox.tsx`
b) The actual checkbox (i.e. the `<input>` is wrapped in another div

Therefore, I think the best choice would be to remove this style here and add `min-width: 20px` after https://github.com/digitalfabrik/integreat-app/blob/7767605ed89e6a336c508fd956b4d751ff42518f/web/src/components/base/Checkbox.tsx#L14.
This should also fix the issue you describe in #3132.